[2025-05-04T19:26:47.709+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_load.load_clients manual__2025-05-04T19:26:34.495143+00:00 [queued]>
[2025-05-04T19:26:47.717+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_load.load_clients manual__2025-05-04T19:26:34.495143+00:00 [queued]>
[2025-05-04T19:26:47.717+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-05-04T19:26:47.727+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_clients> on 2025-05-04 19:26:34.495143+00:00
[2025-05-04T19:26:47.731+0000] {standard_task_runner.py:57} INFO - Started process 3619 to run task
[2025-05-04T19:26:47.733+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'extract_load', 'load_clients', 'manual__2025-05-04T19:26:34.495143+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/extract_load.py', '--cfg-path', '/tmp/tmp8k6ws_oc']
[2025-05-04T19:26:47.734+0000] {standard_task_runner.py:85} INFO - Job 128: Subtask load_clients
[2025-05-04T19:26:47.745+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-04T19:26:47.770+0000] {task_command.py:410} INFO - Running <TaskInstance: extract_load.load_clients manual__2025-05-04T19:26:34.495143+00:00 [running]> on host 773728ee9afd
[2025-05-04T19:26:47.839+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='finbest' AIRFLOW_CTX_DAG_ID='extract_load' AIRFLOW_CTX_TASK_ID='load_clients' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T19:26:34.495143+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-04T19:26:34.495143+00:00'
[2025-05-04T19:26:47.846+0000] {logging_mixin.py:150} INFO - Загружено 1200 строк из /data/clients.csv
[2025-05-04T19:26:47.847+0000] {crypto.py:83} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-05-04T19:26:47.847+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-04T19:26:47.851+0000] {sql.py:375} INFO - Running statement: 
    SELECT column_name 
    FROM information_schema.columns 
    WHERE table_schema = 'staging' AND table_name = 'clients'
    , parameters: None
[2025-05-04T19:26:47.855+0000] {sql.py:384} INFO - Rows affected: 7
[2025-05-04T19:26:47.855+0000] {logging_mixin.py:150} INFO - Столбцы в БД для clients: ['client_id', 'first_name', 'last_name', 'email', 'passport_id', 'registration_date', 'status']
[2025-05-04T19:26:47.856+0000] {logging_mixin.py:150} INFO - Используем столбцы для clients: ['client_id', 'first_name', 'last_name', 'email', 'passport_id', 'registration_date', 'status']
[2025-05-04T19:26:47.858+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-04T19:26:47.862+0000] {sql.py:375} INFO - Running statement: TRUNCATE TABLE staging.clients, parameters: None
[2025-05-04T19:26:47.868+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-04T19:26:48.142+0000] {logging_mixin.py:150} INFO - Данные загружены в staging.clients
[2025-05-04T19:26:48.143+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-04T19:26:48.151+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=extract_load, task_id=load_clients, execution_date=20250504T192634, start_date=20250504T192647, end_date=20250504T192648
[2025-05-04T19:26:48.186+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-05-04T19:26:48.216+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
